packages/@misk/simpleredux/src/utilities.ts

```diff
@@ -80,7 +81,8 @@ export const errorMessage = (error: any) => {
   let code = error.errorCode
   if (!code) {
     code =
-      error.response && error.response.status === 401
+      error.response &&
```

Comment:
so errorCode can be non-null but response could be null? When is that possible? Can you share the interface of error and the API that it exposes?

Reply:
Not a very well defined API. Primarily displays any JSON object passed in. If it is an error from Axios response and has a `response.code` then it will parse out the code and include it in the title of the `<ErrorCalloutComponent/>`. There are many other outstanding bugs / improvements for that component so a more documented / thought out API would be something to work towards.